content/news/2024-12-12-mixxing-up-pen-and-paper.md

```diff
@@ -0,0 +1,49 @@
+# Mixxxing up Pen and Paper
+
+> It is cold and moist. You are deep within the belly of a huge mashine surrounded by overgrowth and broken glass. You know she wants to kill you. After all: you did the same to her. She wants revenge and the only thing keeping you alive is her unstoppable desire to test.
+>
+> The sraching of plates and distant screams of broken turrets surround you. You are alone; ovserved by an all-seeing mashine. Your only chance of survival is to comply.
+>
+> So, testing. Right.
+>
+> You enter the next chamber.
+>
+> "Did you know that people with a bad conciouse get more easily startled by loud nois - " *CHOOOOOOOO*
+>
+> *She is out to kill you.*
+> 
+> The harsh red light of a lazeer illuminates the huge test chamber, when suddenly, a cube drops from the ceiling. But it is not like the others. Every Cube had a Aperture Logo on it. But this one ... a heart? Memories of betraying an old friend flash before your eyes.
+>
+> You get closer, carefully stepping over the deadly laser beam, when you hear a faint sound. It is friendly. It does not want to hurt you.
+>
+> You go closer to the cube.  
+> Is that voice coming from the ... cube?  
+> You crouch next to it.
+>
+> It hums. A small melody. Fihgting against the eary noise around you.
+
+If I had to pick a favorite moment in a video game, it would be this: a perfect exampple of how mixxing and dynamic soundtracks can not only enhancing, but telling a story outright.
```

Comment:
```suggestion
If I had to pick a favorite moment in a video game, it would be this: a perfect exampple of how mixing and dynamic soundtracks can not only enhancing, but telling a story outright.
```

Reply:
Thanks for proof reading, I'd recommend only giving feedback on the general structure and style for now as whole paragraphs could be subject to change :)